moma_demos/piloting_demo/launch/mission.launch

```diff
@@ -1,7 +1,17 @@
 <?xml version="1.0"?>
 <launch>
   <!-- valve model -->
-  <include file="$(find piloting_demo)/launch/valve.launch"/>
+  <include file="$(find piloting_demo)/launch/valve.launch">
+    <!-- param name="x" value="0"/-->
+    <!-- param name="y" value="0"/-->
+    <!-- param name="z" value="0"/-->
+  </include>
+
+  <!-- detection hotspot relative to estimated valve pose -->
+  <node pkg="tf2_ros"
+        type="static_transform_publisher"
+        name="hotspot_broadcaster"
+        args="-2 0 0 0 0 0 valve_gt hotspot" />
```

Comment:
If we change the valve pose this does not change. Can we make it dependent on the valve pose?